faroe/api/src/main/java/org/n52/faroe/Constants.java

```diff
@@ -0,0 +1,19 @@
+package org.n52.faroe;
+
+public final class Constants {
+
+    public static final String KEY = "Key";
+    public static final String TYPE = "Type";
+    public static final String NAME = "Name";
+    public static final String SETTING_DEFINITIONS = "Setting Definitions";
```

Comment:
These are used as JSON properties? Use `lowerCamelCase` for them.

Reply:
These are constants though right? lowerCamelCase would cause checkstyle issues as well

Reply:
I was talking about the string values.